### ISSUE TITLE
#batch_size=1

### ISSUE THREAD

**Issue by greatnameofcug** [junyanz/pytorch-CycleGAN-and-pix2pix#1477]
I found that the cyclegan network model may be more suitable for batchsize=1. Apart from hardware equipment, a larger batchsize may reduce the learning effect, is that so?

**Comment by taesungp** [comment]
Yes. That is what we found too. One reason could be that our datasets are small, so larger batch size leads to faster overfitting.
